ERROR:root:[Errno 17] File exists: 'file1'
